### PR TITLE
Tidied up some docs. Fixed some non-idiomatic usage of return in scal…

### PR DESCRIPTION
**Pull request by sksamuel**
…a code.

### CODE REVIEW

File: flink-scala/src/main/scala/org/apache/flink/api/scala/package.scala [hunk 1]
```diff
@@ -71,12 +71,11 @@ package object scala {
...
     }
   }
+
   def getCallLocationName(depth: Int = 3) : String = {
     val st = Thread.currentThread().getStackTrace()
-    if(st.length < depth) {
-      return "<unknown>"
-    }
-    st(depth).toString
+    if (st.length < depth) "<unknown>"
```
Comment (fhueske) [inline flink-scala/src/main/scala/org/apache/flink/api/scala/package.scala]: Our [code style](http://flink.apache.org/contribute-code.html#code-style) requires curly braces for `if` statements.

Comment (sksamuel) [inline flink-scala/src/main/scala/org/apache/flink/api/scala/package.scala]: added

**Comment by fhueske** [comment]
Wow, that was quick! :-)
Thanks for the PR and the update.
Looks good to merge, IMO

**Comment by tillrohrmann** [comment]
+1 for merging. Thanks for your contribution @sksamuel :-)

**Comment by fhueske** [comment]
Will merge this PR.

**Pull request closed without merging**